inc/Controller.php

```diff
@@ -520,13 +520,28 @@ public function get_like_count( $object_id ) {
 	public function get_like_count_html( $object_id ) {
 		$likes = $this->get_like_count( $object_id );
 
+		if ( 1 === $likes ) {
+			$likes_text = sprintf(
+				/* translators: $s = number of likes */
+				__( '%s like', 'rest-likes' ),
+				$likes
+			);
```

Comment:
```suggestion
			$likes_text = __( 'One like', 'rest-likes' );
```

Die Überprüfung auf 1 erlaubt uns extra einen anderen Text ohne Platzhalter zu werden. Das sollte überall geändert werden.

Reply:
Schau mal hier: https://github.com/wearerequired/rest-likes/pull/148/commits/6aa53e65aa4035482acfc95be38fc7b30c32e051